Workspaces/Base/WorkspaceUI.cs

```diff
@@ -444,6 +451,185 @@ IEnumerator Start()
 			yield return null;
 		}
 
+		void AdjustHandlesAndIcons()
```

Comment:
Still desiring that this would be simplified, so let's discuss. Any possibility this could be done w/ design as opposed to code?